test/test_external.py

```diff
@@ -515,6 +515,8 @@ def test_use_api_name_in_call_method():
 
 
 def test_load_custom_component():
+    from gradio_pdf import PDF  # noqa
```

Comment:
To load a Space with a custom component, the custom component class must be imported in the same Python session